src/lib/coda_base/block_time.ml

```diff
@@ -43,18 +43,27 @@ module Time = struct
 
     let create offset = offset
 
-    let basic =
+    let basic ~logger =
       lazy
-        ( Core_kernel.Time.Span.of_int_sec @@ Int.of_string
-        @@ Unix.getenv "CODA_TIME_OFFSET" )
+        (let offset =
+           match Core.Sys.getenv "CODA_TIME_OFFSET" with
+           | Some tm ->
+               Int.of_string tm
+           | None ->
+               Logger.info logger ~module_:__MODULE__ ~location:__LOC__
```

Comment:
On further thinking, this should be a `debug` log as this will be printed whenever a user starts up their node during the normal usage

Reply:
During normal usage, isn't CODA_TIME_OFFSET not needed? seems like maybe in that case this code wouldn't be executed?

Reply:
The log message appears only when `time_offsets` is in the config and the environment variable is not set.

I changed the code to use a `debug` message.